pkg/opensearch/response_parser.go

```diff
@@ -98,7 +102,74 @@ func (rp *responseParser) getTimeSeries(configuredFields es.ConfiguredFields) (*
 	return result, nil
 }
 
-func processRawDataResponse(res *es.SearchResponse, timeField string, queryRes backend.DataResponse) backend.DataResponse {
+const level = "level"
```

Comment:
where do we use this?

Reply:
It was used once down in line ~300-something. Not sure if you didn't like that since it was only used once? Removed it